### ISSUE TITLE
 '__spreadArray' is not exported from 'tslib'.

### ISSUE THREAD

**Issue by fearless23** [GetStream/react-activity-feed#265]
Simple ui component:
```html
<div style={{ width: '600px', margin: '0 auto' }}>
  <StreamApp 
    apiKey={STREAM_CHAT_APP_KEY} 
    appId={STREAM_CHAT_APP_ID} 
    token={token}
  >
    <StatusUpdateForm />
    <FlatFeed feedGroup="user" notify />
  </StreamApp>
</div>
```
      
gives following error:
error - ./node_modules/react-activity-feed/dist/components/RepostButton.js
Attempted import error: '__spreadArray' is not exported from 'tslib'.

"react-activity-feed": "^1.1.0", tried with v1.0.0 as well

**Comment by fearless23** [comment]
solved by 
```sh
npm i tslib
```

but this should not be the case

**Comment by arnautov-anton** [comment]
Hey, @fearless23! Thank you for opening this issue. Tested [here](https://codesandbox.io/s/festive-nightingale-cxntw) and it does not seem to throw such error for me. Could you please try to remove `node_modules` folder and run `npm i`/`yarn` and let me know if it happens again.

**Comment by mahboubii** [comment]
fixed in v1.1.1
